dotnet/src/VectorDataIntegrationTests/SqlServerIntegrationTests/SqlServerVectorStoreTests.cs

```diff
@@ -263,7 +263,7 @@ public void TimeSpanIsNotSupported()
         string collectionName = GetUniqueCollectionName();
         var testStore = fixture.TestStore;
 
-        Assert.Throws<ArgumentException>(() => testStore.DefaultVectorStore.GetCollection<string, TimeModel>(collectionName));
+        Assert.Throws<NotSupportedException>(() => testStore.DefaultVectorStore.GetCollection<string, TimeModel>(collectionName));
```

Comment:
The test is called `TimeSpanIsNotSupported` so it's a good thing we throw `NotSupportedException` now